### PR TITLE
Add option to flip video

### PR DESCRIPTION
**Pull request by bahar-ap**
## Description

Adding option to flip video

## Screenshots

mosquitto_sub -t '#' -v -h localhost
OR 
mosquitto_sub -t 'camera/flip_video_feed' -v -h localhost

when clicking flip video, mqtt message should be sent :)

### CODE REVIEW

File: client/src/api/common/powerModel.ts [hunk 1]
```diff
@@ -51,13 +51,13 @@ export function usePowerModel() {
     maxHandler,
   );
 
-  const handleAchievedMaxData = useCallback((data: AchievedData) => {
+  const handleAchivedMaxData = useCallback((data: AchievedData) => {
```
Comment (kdav108) [inline client/src/api/common/powerModel.ts]: I get tripped with this every time, but pretty sure the spelling was correct before, i.e. it's `Achieved`.

File: client/src/components/common/camera_settings/OverlaySelection.tsx [hunk 2]
```diff
@@ -42,6 +44,13 @@ export default function OverlaySelection({
     [selectedOverlay, setActiveOverlay],
   );
 
+  /** Flipping Video Feed */
+  function flipVideo() {
+    toast.success('Video feed is flipped!');
+    console.log('yes sir I got to this point');
```
Comment (kdav108) [inline client/src/components/common/camera_settings/OverlaySelection.tsx]: Glad to know 😂

File: client/src/components/common/camera_settings/OverlaySelection.tsx [hunk 3]
```diff
@@ -42,6 +44,13 @@ export default function OverlaySelection({
     [selectedOverlay, setActiveOverlay],
   );
 
+  /** Flipping Video Feed */
+  function flipVideo() {
+    toast.success('Video feed is flipped!');
```
Comment (kdav108) [inline client/src/components/common/camera_settings/OverlaySelection.tsx]: Maybe make this toast appear after the `emit` call, so if that errors out we don't incorrectly send a success toast message.

**Review by kdav108 — CHANGES_REQUESTED** [review]
Nice short PR! Nothing to complain here, just a few nit picks 😂.

Also, it seems you've branched off from `bahar-ap/change_achieved_max_speed_name` instead of `master`, that's fine, just make sure you merge #135 first and then update this branch before merging this :)

File: client/src/components/common/camera_settings/OverlaySelection.tsx (hunk 2 above)
Comment (bahar-ap) [reply client/src/components/common/camera_settings/OverlaySelection.tsx]: whoops!!! 😅

File: server/js/sockets.js [hunk 4]
```diff
@@ -375,6 +375,11 @@ sockets.init = function socketInit(server) {
       mqttClient.publish(Camera.recording_stop);
     });
 
+    socket.on('flip-video-feed', () => {
+      console.log('got to this point too sir no problem');
```
Comment (kdav108) [inline server/js/sockets.js:379]: You forgot to remove this as well 😂

File: client/src/components/v3/StatisticRow.tsx [hunk 5]
```diff
@@ -26,13 +26,17 @@ export default function StatisticRow(): JSX.Element {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currVel]);
 
+  // Previous Trap Speed (Achieved)
+  // eslint-disable-next-line no-unused-vars
```
Comment (kdav108) [inline client/src/components/v3/StatisticRow.tsx:30]: we don't require this `ESLint-disable-next-line` stuff cos we now do use the `setPrevTrapSpeed` func. Previously we had it defined but never called it and hence we got the `no-unused-vars` error from ESLint, and had to disable it.

File: client/src/components/v3/StatisticRow.tsx [hunk 6]
```diff
@@ -26,13 +26,17 @@ export default function StatisticRow(): JSX.Element {
...
   const [prevTrapSpeed, setPrevTrapSpeed] = useState<number | null>(null);
 
   const handlePrevTrapSpeed = useCallback((payload: SpeedPayload) => {
     setPrevTrapSpeed(payload.speed);
   }, []);
   useChannelShaped('boost/prev_trap_speed', SpeedPayload, handlePrevTrapSpeed);
 
+  // Next Trap Speed (Predicted)
+  // eslint-disable-next-line no-unused-vars
```
Comment (kdav108) [inline client/src/components/v3/StatisticRow.tsx:39]: Same here, should not be needed anymore

**Review by kdav108 — APPROVED** [review]
We're done, just really minor stuff to get rid of.

I'll approve still, we can merge after the comments below are resolved.

**Review by hallgchris — APPROVED** [review]
Nothing to add! In the future it would be great if you could include a screenshot in the PR description!

**Pull request merged**